x/twap/api.go

```diff
@@ -8,15 +8,14 @@ import (
 	"github.com/osmosis-labs/osmosis/v13/x/twap/types"
 )
 
-type twapType bool
-
-const (
-	// arithmeticTwapType is the type of twap that is calculated by taking the arithmetic weighted average of the spot prices.
-	arithmeticTwapType twapType = true
-	// geometricTwapType is the type of twap that is calculated by taking the geometric weighted average of the spot prices.
-	// nolint: unused
-	geometricTwapType twapType = false
-)
+type twapStrategies twapStrategy
+
+// arithmeticTwapType is the type of twap that is calculated by taking the arithmetic weighted average of the spot prices.
+var arithmeticTwapStrategy twapStrategies = &arithmetic{}
```

Comment:
We are using `arithmeticTwapStrategy` and `geometricTwapStrategy` in `export_test.go` like this (https://github.com/osmosis-labs/osmosis/pull/3696/files#diff-51efcb283ad8f93cd90845c8f79784fdacad04fe78ffecf80fa934ec25f3a330R16)

Reply:
The test should import types, not variables. Given a type, the test can initialize each strategy.

Here, the initialization is invalid because each strategy has a keeper as a field. We should not add initializations in production code only for the purposes of importing variables in tests